deploy/kubernetes/metricbeat/README.md

```diff
@@ -2,30 +2,145 @@
 
 ## Ship metrics from Kubernetes to Elasticsearch
 
-### Kubernetes DaemonSet
+### Settings
 
-By deploying metricbeat as a [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/)
-we ensure we get a running metricbeat daemon on each node of the cluster.
+We use official [Metricbeat Docker images](https://www.docker.elastic.co/r/beats/metricbeat), as they allow external files' configuration. Our YAML manifests are the following:
+- [metricbeat-daemonset-configmap.yaml](metricbeat-daemonset-configmap.yaml) to create the [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/) necessary to [metricbeat-daemonset.yaml](metricbeat-daemonset.yaml).
+- [metricbeat-daemonset.yaml](metricbeat-daemonset.yaml) to create a [DeamonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/). This way we ensure we get a running metricbeat daemon on each node of the cluster.
+  This files uses a set of environment variables to configure Elasticsearch output.
 
-Everything is deployed under `kube-system` namespace, you can change that by
-updating YAML manifests under this folder.
+| Variable               | Default       | Description                          |
+|------------------------|---------------|--------------------------------------|
+| ELASTICSEARCH_HOST     | elasticsearch | Elasticsearch host                   |
+| ELASTICSEARCH_PORT     | 9200          | Elasticsearch port                   |
+| ELASTICSEARCH_USERNAME | elastic       | Elasticsearch username for HTTP auth |
+| ELASTICSEARCH_PASSWORD | changeme      | Elasticsearch password               |
```

Comment:
could you please explain that these are two sets of variables either host, port, username and password for managed installation or cloud_id and cloud_auth for cloud installation. 
It would be great also to add a link to both sets of configs either [standalone](https://www.elastic.co/guide/en/beats/metricbeat/current/elasticsearch-output.html) or [managed/cloud](https://www.elastic.co/guide/en/beats/metricbeat/current/configure-cloud-id.html)